setup.py

```diff
@@ -1,43 +1,68 @@
 #!/usr/bin/env python
+import csv
+import imp
 import os
 import shutil
 import sys
 from setuptools import setup, find_packages
 
-VERSION = '0.0.1'
+DESCRIPTION = 'Wavelet scattering transforms in Python with GPU acceleration'
+VERSION = imp.load_source('kymatio.version', 'kymatio/version.py').version
+with open('README.md') as f:
+    LONG_DESCRIPTION = f.read()
+URL = 'https://kymatio.github.io'
+LICENSE = 'BSD-3-Clause'
 
-long_description = """
-Fast CPU/CUDA Scattering Transform implementation for 1D, 2D, and 3D signals
 
-CuPy/PyTorch CUDA and NumPy/PyTorch CUDA implementation
-"""
+# Parse requirements.txt
+with open('requirements.txt', 'r') as f:
+    REQUIREMENTS = [row[0] for row in csv.reader(f, delimiter='\n')]
```

Comment:
Wouldn't a `f.read().split("\n")` suffice?

It's not really csv format. Does the `csv` reader somehow guarantee some form of stability or something?

Reply:
a `csv` reader was the first thing that came to mind
I should have thought about `f.read()`

Reply:
fixed in ff00761